Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -541,4 +543,305 @@ void doEsirkepovDepositionShapeN (const GetParticlePosition& GetPosition,
         );
 }
 
+/**
+ * \brief Vay current deposition for thread thread_num: deposit D in real space
+ * (equations (21)-(22) of https://doi.org/10.1016/j.jcp.2013.03.010)
```

Comment:
Could you add a comment
```suggestion
 * (equations (21)-(22) of https://doi.org/10.1016/j.jcp.2013.03.010)
 * Store the result in \c jx_fab, \c jy_fab, \c jz_fab
```